src/kirby/components/icon/icon.component.html

```diff
@@ -1 +1 @@
-<ion-icon [src]="icon?.svg"></ion-icon>
+<ion-icon [src]="icon?.svg" [ngClass]="{'outline': outline}"></ion-icon>
```

Comment:
Consider simplifying:
```suggestion
<ion-icon [src]="icon?.svg" [class.outline]="outline"></ion-icon>
```